lib/spack/docs/packaging_guide.rst

```diff
@@ -4048,6 +4048,69 @@ File functions
 :py:func:`touch(path) <spack.touch>`
   Create an empty file at ``path``.
 
+.. _make-package-findable:
+
+--------------------------------------------------------
+Making a package discoverable with "spack external find"
+--------------------------------------------------------
+
+To make a package discoverable with ``spack external find`` you must
```

Comment:
`spack external find` here should be a link back to the build settings docs (so you can easily get to usage of the command if you don't know what this is about)